distribution_rounds/round-1/distribution_results/raw_csv_exports/forum_post.md

```diff
@@ -0,0 +1,106 @@
+
+# TEC Rewards Distribution - round-1  - 11/07/21 to 30/07/21
+This period covers praise given between 11. July 2021 and 30. July 2021. We allocated 2546 TEC tokens for rewards, with a 100:0 split between Praise and Sourcecred. Some praise accounts still haven’t been activated so the total amount below will be less than what we set aside to distribute.
+
+Out of the total rewards:
+
+* 90% of the tokens were given as praise rewards :pray:
+* 7% distributed among quantifiers :balance_scale:
+* 3% assigned to the reward board :memo:
+
+This data has been reviewed by the Quantifiers and the Reward Board, and has been submitted for distribution to the [Reward Board DAO 1](https://xdai.aragon.blossom.software/#/rewardboardtec/)
```

Comment:
you have included accidentally the link click counter from the forum -  `[Reward Board DAO 1]` should be [Reward Board DAO]